torch/nn/quantized/modules/linear.py

```diff
@@ -61,3 +61,17 @@ def forward(self, x):
             self.out_scale,
             self.out_zero_point)
         return Y_q
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
+        destination[prefix + 'weight'] = torch.ops.quantized.fbgemm_linear_unpack(destination[prefix + '_packed_weight'])
+        destination.pop(prefix + '_packed_weight')
+
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        self._packed_weight = torch.ops.quantized.fbgemm_linear_prepack(state_dict[prefix + 'weight'])
+        self.bias = state_dict[prefix + 'bias']
```

Comment:
it's better to preserve the identity of the bias buffer, so do `self.bias.copy_(state_dict[prefix + 'bias'])`?